it/uk/gov/hmrc/excisemovementcontrolsystemapi/DraftExciseMovementControllerItSpec.scala

```diff
@@ -124,6 +124,36 @@ class DraftExciseMovementControllerItSpec extends PlaySpec
       postRequest(IE815).status mustBe BAD_REQUEST
     }
 
+    "remove control document references in any paths for a BAD_REQUEST" in {
+      val eisErrorResponse = "{" +
+        "\"message\": \"Validation error(s) occurred\"," +
+        "\"errors\": [" +
+        "{\"errorCode\": 8084," +
+        "\"errorMessage\": \"The Date of Dispatch you entered is incorrect. It must be today or later. Please amend your entry and resubmit.\"," +
+        "\"location\": \"/con:Control[1]/con:OperationRequest[1]/con:Parameters[1]/con:Parameter[1]/urn:IE815[1]/urn:Body[1]/urn:SubmittedDraftOfEADESAD[1]/urn:EadEsadDraft[1]/urn:DateOfDispatch[1]\"," +
+        "\"value\": \"2023-12-05\"}" +
+        "]" +
+        "}"
+
+      val apiErrorResponse = "{\"message\": \"Validation error(s) occurred\",\"errors\": [" +
```

Comment:
do you think this could go outside the test, maybe as private val at the top of the class or as function, so we make the test shorter and less busy?

Reply:
done